plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/internal/EditorsResources_de.properties

```diff
@@ -44,9 +44,9 @@ dialog_morph_delimited_target_group_delim_result = Ergebnis Trennzeichen
 dialog_morph_delimited_target_group_leading_text = F\u00FChrender Text
 dialog_morph_delimited_target_group_trailing_text = Nachlaufender Text
 
-database_editor_command_save_name = Speichern ...
-database_editor_command_save_tip = Vorschau\u00E4nderungen und speichern
+database_editor_command_save_name = Speichern
```

Comment:
I don't see the reason to remove `...` from `Save...` command because `...` indicates continuation of dialog.